web/packages/teleterm/src/ui/Search/pickers/ParameterPicker.tsx

```diff
@@ -55,10 +56,10 @@ export function ParameterPicker(props: ParameterPickerProps) {
         { value: inputValue, displayText: inputValue },
       ]
   );
-  const $suggestionsError =
-    suggestionsAttempt.status === 'error' ? (
-      <SuggestionsError statusText={suggestionsAttempt.statusText} />
-    ) : null;
+  const $suggestionsMessage = getSuggestionsMessage({
+    suggestionsAttempt,
```

Comment:
The check must be done against `attempt`, not `suggestionsAttempt`, so that the message is shown also when you type in a string that doesn't match any value.

It might be worthwhile to rename `attempt` to `matchingSuggestionsAttempt` or something like that.

Reply:
Ah right, thanks.